library/panic_unwind/src/lib.rs

```diff
@@ -25,13 +25,14 @@
 // `real_imp` is unused with Miri, so silence warnings.
 #![cfg_attr(miri, allow(dead_code))]
 #![allow(internal_features)]
+#![cfg_attr(not(bootstrap), feature(cfg_emscripten_wasm_eh))]
```

Comment:
@Urgau How do I allow using this feature in the stage0 build? This `not(bootstrap)` makes the stage0 stdlib build work, but once this commit is merged the stage0 build will be broken.

Reply:
You don't need to allow using the feature in `stage0` as `stage0` is the beta compiler and it doesn't know about the feature.

When `stage0` will be updated the `#[cfg_attr]` condition will be removed leaving only `#![feature(cfg_emscripten_wasm_eh)]`, which will be known at that point by `stage0`.

Reply:
Thanks for the explanation.